src/functions.ts

```diff
@@ -21,7 +22,8 @@ function greet(name: string): void {
  */
 function isOdd(n: number): boolean {
   // Your code here
-
+  if (n%2 != 0)
+    return true;
   return false; // replace false with what you see is fit
 }
```

Comment:
You can just return the result of the condition:

```ts
function isOdd(n: number): boolean {
  return n%2 !== 0;
}
```

Also, please use `===` and `!==`